### DISCUSSION TITLE
Search & Move button

### DISCUSSION THREAD

**Discussion by Amarjeet664** [pimcore/pimcore#15945] [Q&A / Support]
We have a **Search & Move** button on folders but the functionality is working unexpectedly.
No console errors or API calls are being made.

Functionality not working correctly on Pimcore v11.0.8 but working fine on demo instance

![image](https://github.com/pimcore/pimcore/assets/21308395/237af8ce-9c7a-4476-9da8-21d550d7936b)

**Comment by aryaantony92** [comment]
@Amarjeet664 Thanks for reporting the issue. We will look into it. https://github.com/pimcore/pimcore/issues/15996

**Comment by aryaantony92** [comment]
@Amarjeet664 Could you please check whether 'pimcore_search_backend_message' is added to your supervisord ? https://github.com/kaurov/skeleton/blob/c6310909fbbb9c0fc168a480ddfaabef84bb3285/README.md?plain=1#L44
